src/Plug.vala

```diff
@@ -92,7 +92,8 @@ public class ApplicationsPlug : Switchboard.Plug {
     }
 
     public override async Gee.TreeMap<string, string> search (string search) {
-        var search_results = new Gee.TreeMap<string, string> ((GLib.CompareDataFunc<string>)strcmp, (Gee.EqualDataFunc<string>)str_equal);
+        var search_results = new Gee.TreeMap<string, string> ((GLib.CompareDataFunc<string>)strcmp,
+                                (Gee.EqualDataFunc<string>)str_equal);
```

Comment:
This would probably look nicer as:
```
var search_results = new Gee.TreeMap<string, string> (
    (GLib.CompareDataFunc<string>)strcmp,
	(Gee.EqualDataFunc<string>)str_equal
);
```